src/AWSS3.jl

```diff
@@ -493,24 +541,24 @@ function s3_list_objects(
     bucket,
     path_prefix="";
     delimiter="/",
+    start_after="",
     max_items=nothing,
     kwargs...,
 )
     return Channel() do chnl
         more = true
         num_objects = 0
-        marker = ""
+        token = ""
 
         while more
             q = Dict{String,String}()
-            if path_prefix != ""
-                q["prefix"] = path_prefix
-            end
-            if delimiter != ""
-                q["delimiter"] = delimiter
-            end
-            if marker != ""
-                q["marker"] = marker
+            for (v, name) in [
+                (path_prefix, "prefix"),
+                (delimiter, "delimiter"),
+                (start_after, "start-after"),
+                (token, "continuation-token"),
+            ]
+                isempty(v) || (q[name] = v)
```

Comment:
`(key, value)` order makes it much easier to parse for me:
```suggestion
            for (name, v) in [
                ("prefix", path_prefix),
                ("delimiter", delimiter),
                ("start-after", start_after),
                ("continuation-token", token),
            ]
                isempty(v) || (q[name] = v)
```